src/cdcacm_serial.c

```diff
@@ -89,9 +89,7 @@ static void write_data(struct device *dev, const char *buf, int len)
 	uart_fifo_fill(dev, (const uint8_t*)buf, len);
 	while (data_transmitted == false)
```

Comment:
what are the implications of removing this yield() call? does it mean that x86 will be stuck busy waiting in this thread until the whole string is written?